packages/app/src/specs/spec-utils.ts

```diff
@@ -1,14 +1,20 @@
 import fuzzySort from 'fuzzysort'
 import type { FoundSpec } from '@packages/types'
 import { ComputedRef, Ref, ref, watch } from 'vue'
-import _ from 'lodash'
+import _, { cloneDeep } from 'lodash'
 import { FuzzyFoundSpec, getPlatform } from './tree/useCollapsibleTree'
 
 export function fuzzySortSpecs <T extends FuzzyFoundSpec> (specs: T[], searchValue: string) {
-  const normalizedSearchValue = getPlatform() === 'win32' ? searchValue.replaceAll('/', '\\') : searchValue
+  const normalizedSearchValue = normalizeSpecName(searchValue)
+
+  const clonedSpecs = cloneDeep(specs).map((spec) => {
```

Comment:
Instead of cloning the specs, I think it would make more sense to add two new properties (`normalizedBaseName` and `normalizedRelative`) to `FuzzyFoundSpec` and populate them in the `makeFuzzyFoundSpec` function.